vpr/src/route/rr_graph_storage.cpp

```diff
@@ -200,6 +200,14 @@ class edge_sort_iterator {
         return lhs.swapper_.idx_ < rhs.swapper_.idx_;
     }
 
+    friend bool operator>(const edge_sort_iterator& lhs, const edge_sort_iterator& rhs) {
+        return lhs.swapper_.idx_ > rhs.swapper_.idx_;
+    }
+
+    friend bool operator>=(const edge_sort_iterator& lhs, const edge_sort_iterator& rhs) {
```

Comment:
While we are here, does it make sense to define <=?

Reply:
Can't harm. Added.